### DISCUSSION TITLE
How to use it with Nuxt 3.13 with nuxt-electron and @nuxtjs/ionic plugins

### DISCUSSION THREAD

**Discussion by RichySano** [capacitor-community/sqlite#597] [General]
Hi,

I want to use this capacitor with Nuxt v3.13 with nuxt-electron and @nuxtjs/ionic plugins.

So i use the last version of Electron 33.0.2.

When i use this capacitor in main process of electron i've the following error :

SQLite Error: Error: The jeep-sqlite element is not present in the DOM! Please check the @capacitor-community/sqlite documentation for instructions regarding the web platform.
    at i.ensureJeepSqliteIsAvailable (file:///C:/Users/richa/OneDrive/Bureau/test-electron/dist-electron/web-DVZPEbOx.js:315:13)
    at i.echo (file:///C:/Users/richa/OneDrive/Bureau/test-electron/dist-electron/web-DVZPEbOx.js:47:17)
    at file:///C:/Users/richa/OneDrive/Bureau/test-electron/dist-electron/main-rBiB6DTB.js:66:23
    at async file:///C:/Users/richa/OneDrive/Bureau/test-electron/dist-electron/main-rBiB6DTB.js:380:15

I don't want to use capacitor-community/electron which use an older version of electron.

My project is already running on Web, Electron, Android and Iphone but i want to use SQLite.

If you can guide me in the way to follow

Thank ou

**Comment by jepiqueau** [comment]
@RichySano You must use @capacitor-community/electron to get the @capacitor-community/sqlite plugin with Electron. I do not think it will be an issue to update the @capacitor-community/electron  to Electron 33.0.2. Have you tried it?

**Reply by GYSRichardGUION** [reply]
@jepiqueau, thanks for your response but why in readme it's indicated

On Electron, go to the Electron folder of YOUR_APPLICATION
```bash
cd electron
npm install --save better-sqlite3-multiple-ciphers
npm install --save electron-json-storage
npm install --save jszip
npm install --save node-fetch@2.6.7
npm install --save crypto
npm install --save crypto-js
npm install --save-dev @types/better-sqlite3
npm install --save-dev @types/electron-json-storage
npm install --save-dev @types/crypto-js
```
Important: node-fetch version must be <=2.6.7; otherwise https://github.com/capacitor-community/sqlite/issues/349 running the app.

Important: if you are using @capacitor-community/electron v5

you have to stick to Electron@25.8.4...

Implicitly, this indicates that we can do without @capacitor-community/electron

Yes, I am using the latest version of electron 33.0.2

RichySano is my personal account and GYSRichardGUION is my professional account
